packages/cli/src/workflows/workflows.controller.ts

```diff
@@ -460,4 +461,16 @@ export class WorkflowsController {
 			workflow,
 		});
 	}
+
+	@Put('/:workflowId/transfer')
+	@ProjectScope('workflow:move')
```

Comment:
Not sure how this validation would work - would it check the user's personal project or the target project scope?